ibis/expr/datatypes.py

```diff
@@ -778,6 +808,7 @@ def valid_literal(self, value):
 date = Date()
 time = Time()
 timestamp = Timestamp()
+interval = Interval()
```

Comment:
Do we want aliases for each of the interval values like `year`, `month`, `day`, etc.?

Reply:
Fine if not, just a thought.

Reply:
I was just following the patterns in datatypes.py
None of the parametric types has specified aliases, so I guess not.